skimage/feature/_cascade.pyx

```diff
@@ -909,7 +909,7 @@ cdef class Cascade:
                 feature_number = int(internal_nodes[0])
                 # list() is for Python3 fix here
                 lut_array = list(map(lambda x: int(x), internal_nodes[1:]))
-                lut = np.asarray(lut_array, dtype='uint32')
+                lut = np.array(lut_array).astype(np.uint32)
```

Comment:
why is this change important?

Reply:
See https://github.com/scikit-image/scikit-image/issues/6638

I think perhaps, this suggestion is better than just ignoring the warning as I currently have in #6637. I think I may just update that one with a co-authored-by commit crediting @bmorris3 as this will continue to work even after the deprecation cycle completes